### PR TITLE
Implementing API for taking action

### PR DESCRIPTION
**Pull request by emailweixu**

### CODE REVIEW

File: social_bot/test.py [hunk 1]
```diff
@@ -1,15 +1,17 @@
 # Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
 
 import social_bot as bot
-bot.initialize()
-world = bot.new_world_from_file("animated_box.world")
+bot.initialize(["abc", "--verbose"])
```
Comment (bayesian) [inline social_bot/test.py]: no need for those args?

File: social_bot/social_bot.cc [hunk 2]
```diff
@@ -44,7 +78,10 @@ class World {
 void Initialize(const std::vector<std::string>& args) {
   gazebo::common::Console::SetQuiet(false);
   static std::once_flag flag;
-  std::call_once(flag, [&args]() { gazebo::setupServer(args); });
+  std::call_once(flag, [&args]() {
+    gazebo::common::Console::SetQuiet(false);
```
Comment (bayesian) [inline social_bot/social_bot.cc:81]: remove this line?

**Review by bayesian — CHANGES_REQUESTED** [review]

File: social_bot/social_bot.cc (hunk 2 above)
Comment (emailweixu) [reply social_bot/social_bot.cc:81]: I'll remove line 79

File: social_bot/test.py (hunk 1 above)
Comment (emailweixu) [reply social_bot/test.py]: will remove

File: social_bot/social_bot.cc (hunk 2 above)
Comment (bayesian) [reply social_bot/social_bot.cc:81]: ok

File: social_bot/test.py [hunk 3]
```diff
@@ -2,14 +2,16 @@
...
+agent.get_joint_names()
+agent.take_action({"pioneer2dx::pioneer2dx::right_wheel_hinge": 10.})
+
+for i in range(1000000):
     # observation = agent.sense()
     # add reward and text to observation
     # action = model.compute_action(observation)
     # agent.take_action(action)
     world.step()
-    raw_input("Press Enter to step")
```
Comment (bayesian) [inline social_bot/test.py:15]: my bad.. input or raw_input has different behavior in python2 vs python3.

**Review by bayesian — APPROVED** [review]

**Pull request merged**